x/migration/module/autocli.go

```diff
@@ -68,6 +68,13 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 					PositionalArgs: []*autocliv1.PositionalArgDescriptor{{ProtoField: "morseSrcAddress"}, {ProtoField: "morseSignature"}, {ProtoField: "stake"}, {ProtoField: "serviceConfig"}},
 					Skip:           true, // skipped because autoCLI cannot handle loading & signing using the Morse key.
 				},
+				{
+					RpcMethod:      "RecoverMorseAccount",
+					Use:            "recover-morse-account [shannon-dest-address] [morse-src-address]",
+					Short:          "Send a recover_morse_account tx",
```

Comment:
Leave a TODO to have examples and a long description - this part is super useful.